src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/ReconciliationControllerTest.java

```diff
@@ -35,6 +38,7 @@ void should_return_success_when_supplier_statement_report_is_valid() throws Exce
         mockMvc
             .perform(
                 post("/reform-scan/reconciliation-report/2020-08-10")
+                    .header(HttpHeaders.AUTHORIZATION, "valid-api-key")
```

Comment:
should include Bearer prefix ?

Reply:
do we still need `Bearer` prefix? we decided to use Base64 encoded uuid instead of jwt token

Reply:
I'll update the tests and validation method to include `Bearer` prefix